EIPS/eip-future_salaries.md

```diff
@@ -0,0 +1,71 @@
+
+2. This would effectively decrease ETH supply. The necessity to decrease the ETH supply (by locking ETH in Future Salaries system for a long time) is a well-known important things to be done.
+
+> For a multi-asset smart contract platform to function as a store of value, proper incentives must be put in place to align in the growth in value of a network's assets with its underlying security. Or put another way, the platform's native token must be a good value capture of the platform's aggregate asset value. If the intrinsic value of a platform's native token is limited to transaction fee payment, its value would be determined solely by transaction demand, instead of the demand of asset storage. (https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0001-positioning/0001-positioning.md)
+
+In other words, we need to measure the ETH "real" value. This EIP proposes to allocate *much* (it probably may be more than 90%) of miner's ETH fees to Future Salaries contract. So the cost of ETH will be associated with the real ("job") value of the conditional tokens of this contract.
+
+So, this proposal partly solves both the well-known problems of
+- decreasing ETH supply
+- making ETH cost proportionally increasing when tokens' value increases (in other word incresing the ETH intristic value)
+
+As the above mentioned RFC explains, it is good for miners/validators, because:
+- It reduces manipulation incentives such as selfish mining
+- It highly increases the security of the Ethereum network
+- It increases the ETH intristic value, so miners/validators spend less on energy per amount of money earned.
+- There is the danger (https://github.com/nervosnetwork/rfcs/blob/master/rfcs/0001-positioning/0001-positioning.md) that without radical measures like this proposal the value of ETH (not of Ethereum tokens) will become zero in some future, so greatly downshifting miners/validators.
```

Comment:
Consider shortening this section a bit.  The motivation section should give a very high level overview of why this EIP is desirable, but it should not be an exhaustive argument in favor of it.  The shorter it is, the more likely people are to actually read it.  Additional argumentation/debate/content can live elsewhere, outside of the EIP.  Remember, EIPs are just technical standards and the motivation section is merely to give readers a very brief introduction into the EIP's origins.